site/content/docs/5.1/components/title-bars.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: docs
+title: Title bars
+description: Documentation and examples for Boosted's exclusive Brand responsive title bars.
+group: components
+toc: true
+---
+
+## How it works
+
+A title bar is designed to display the title of a page and shall be placed right below an [Orange navbar]({{<docsref "/components/orange-navbar">}}).
+
+Being based on spacing and backgrounds utilities, title bars were designed to take `100vw` width.
+
+See them in action in our [examples page]({{<docsref "/examples/title-bars">}}).
+
+## Background color
+
+To change the appearance of a title bar, use our [background utilities]({{<docsref "/utilities/background">}}). Orange branded background utilities are `.bg-dark`, `.bg-white` and `.bg-supporting-*`.'
+
+Title bars can be displayed with black or white backgrounds. When using black or white backgrounds, illustrations can be included as an option, but it's not necessary.
+Supporting colors can also be used as the background color for title bars, but not the supporting light and dark tint colors. When using one of the supporting colors, illustrations must be used.
+
+The image should not overlap on title and the title shouldn't wrap. If this happens, please **transform your title bar** into a white/black one without any image.
+
+{{< example class="p-0">}}
+<div class="bg-white title-bar">
+  <div class="container-xxl">
+    <h1 class="display-1">Title</h1>
+  </div>
+</div>
+
+<div class="mt-3 bg-body"></div>
+
+<div class="bg-dark title-bar">
+  <div class="container-xxl">
+    <h1 class="display-1">Title</h1>
+  </div>
+</div>
+
+<div class="mt-3 bg-body"></div>
```

Comment:
`.mt-3` is enough, I think you can remove those `.bg-body`s everywhere when used for separators.